config/config.php

```diff
@@ -36,11 +36,22 @@
 /**
  * Hooks
  */
-$GLOBALS['TL_HOOKS']['parseArticles'][] = array('News', 'addCategoriesToTemplate');
-$GLOBALS['TL_HOOKS']['replaceInsertTags'][] = array('News', 'parseCategoriesTags');
+$GLOBALS['TL_HOOKS']['parseArticles']['news_categories']     = ['News', 'addCategoriesToTemplate'];
+$GLOBALS['TL_HOOKS']['replaceInsertTags']['news_categories'] = ['News', 'parseCategoriesTags'];
+$GLOBALS['TL_HOOKS']['replaceInsertTags']['news_categories'] = ['InsertTags', 'replace'];
 
-if (in_array('changelanguage', \ModuleLoader::getActive())) {
-    $GLOBALS['TL_HOOKS']['translateUrlParameters'][] = array('NewsCategories', 'translateUrlParameters');
+
+if (in_array('changelanguage', \ModuleLoader::getActive()))
+{
+    $GLOBALS['TL_HOOKS']['translateUrlParameters'][] = ['NewsCategories', 'translateUrlParameters'];
+}
+
+foreach ($GLOBALS['TL_HOOKS']['getSearchablePages'] as &$arrCallback)
+{
+    if ($arrCallback[0] == 'News' && $arrCallback[1] == 'getSearchablePages')
+    {
+        $arrCallback = ['News', 'getSearchablePages'];
```

Comment:
Yeah but here you do not change anything, the resulting hooks array will be exactly the same as you replace `['News', 'getSearchablePages']` array with the very same `['News', 'getSearchablePages']` 😉  Nevertheless it's not a good idea to overwrite the core classes... needs to be changed in 3.0.